src/Symfony/Component/Validator/composer.json

```diff
@@ -42,9 +42,6 @@
         "symfony/property-access": "For using the 2.4 Validator API",
         "symfony/expression-language": "For using the 2.4 Expression validator"
     },
-    "conflict": {
-        "egulias/email-validator": ">=2.0"
```

Comment:
Shouldn't this conflict still exist for the versions that are incompatible, or are all versions now compatible in this version of Symfony that's going to be released?

Reply:
This is the same as it is all currently maintained versions: The `EmailValidator` class checks if the installed version can be used when the `strict` option is enabled.